### ISSUE TITLE
"<gapit> Waiting for connection to localhost" can not be cancelled

### ISSUE THREAD

**Issue by dsrbecky** [google/gapid#1241]
Pressing Ctrl+C has no effect.

**Comment by ben-clayton** [comment]
But you **hate** cancellation, right? :)

**Comment by dsrbecky** [comment]
Nah, I think cancellation is like multi-threading...  ok if it is used with care where really needed and with coarse granularity, but I hate when it proliferates into every little method.
